src/test/java/io/pravega/connectors/flink/FlinkPravegaReaderTest.java

```diff
@@ -314,7 +314,7 @@ public long extractTimestamp(Integer element, long previousElementTimestamp) {
             expected.add(record(2, 2));
             expected.add(watermark(2));
             // automatic watermark at the end of stream
-            expected.add(watermark(Long.MAX_VALUE));
+            //expected.add(watermark(Long.MAX_VALUE));
```

Comment:
This is because of https://github.com/apache/flink/commit/3171edfe78a7f9feea88488273bf4341eee7b7f0 I think we can remove the line.